pkg/controller/identity/iamgroupusermembership/controller.go

```diff
@@ -93,16 +94,27 @@ func (e *external) Observe(ctx context.Context, mgd resource.Managed) (managed.E
 		return managed.ExternalObservation{}, errors.New(errUnexpectedObject)
 	}
 
+	if meta.GetExternalName(cr) == "" {
+		return managed.ExternalObservation{ResourceExists: false}, nil
+	}
+
+	// We need to use external name if it exists in order to support resource import.
+	nn := strings.Split(meta.GetExternalName(cr), "/")
+	if len(nn) != 2 {
+		return managed.ExternalObservation{}, errors.New("external name has to be in the following format <group-name>/<user-name>")
+	}
+	groupName, userName := nn[0], nn[1]
```

Comment:
My choice not to support using the external name for importing resources was conscious - I'm not sure this change makes sense. Aren't we now duplicating information from the spec (i.e. the group and user name) in the external name rather than _moving_ those fields into the external name? I don't think there's an obvious way to move them into the EN wholesale since they're both set by references.